lib/msh3.hpp

```diff
@@ -243,16 +243,14 @@ inline QUIC_STREAM_OPEN_FLAGS ToQuicOpenFlags(MSH3_REQUEST_FLAGS Flags) {
 }
 
 inline QUIC_SEND_FLAGS ToQuicSendFlags(MSH3_REQUEST_FLAGS Flags) {
-    QUIC_SEND_FLAGS QuicFlags = QUIC_SEND_FLAG_NONE;
+    QUIC_SEND_FLAGS QuicFlags = QUIC_SEND_FLAG_START;
```

Comment:
I wonder if we should add a new flag in MsQuic: `QUIC_SEND_FLAG_START_IMMEDIATE`.